sections/dir.zsh

```diff
@@ -11,6 +11,7 @@ SPACESHIP_DIR_SHOW="${SPACESHIP_DIR_SHOW:=true}"
 SPACESHIP_DIR_PREFIX="${SPACESHIP_DIR_PREFIX:="in "}"
 SPACESHIP_DIR_SUFFIX="${SPACESHIP_DIR_SUFFIX:="$SPACESHIP_PROMPT_DEFAULT_SUFFIX"}"
 SPACESHIP_DIR_TRUNC="${SPACESHIP_DIR_TRUNC:=3}"
+SPACESHIP_DIR_TRUNC_PREFIX="${SPACESHIP_DIR_TRUNC_PREFIX:=.../}"
```

Comment:
Just not to forget, when you rebase on latest `3.0` this line should become `SPACESHIP_DIR_TRUNC_PREFIX="${SPACESHIP_DIR_TRUNC_PREFIX=".../"}"`